lints/lint_issuer_dn_country_not_printable_string.go

```diff
@@ -0,0 +1,51 @@
+//lint_issuer_dn_country_not_printable_string.go
+package lints
+
+import (
+	"encoding/asn1"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+)
+
+type IssuerDNCountryNotPrintableString struct{}
+
+func (l *IssuerDNCountryNotPrintableString) Initialize() error {
+	return nil
+}
+
+func (l *IssuerDNCountryNotPrintableString) CheckApplies(c *x509.Certificate) bool {
+	return len(c.Issuer.Country) > 0
```

Comment:
Is it technically possible to have a 0-length `Issuer.Country` here but have a field that is not a `PrintableString` without an error?

Reply:
If I have understood the question correctly, then yes. 
As far as code organisation goes here, every lint have a very limited scope. So I didn't check here for empty sequences. 
There is a check if the Country field is valid (value is on a list of country codes) and this check should fire off if the field itself is empty. I am not sure if that lint verifies both issuer country and subject country or just subject country.

Reply:
Okay, that makes sense.